cmd/service/main.go

```diff
@@ -28,6 +33,32 @@ func startSignalHandler(c context.Context) (context.Context, context.CancelFunc)
 	return ctx, cancel
 }
 
+// initRbacWrapper initialize the client wrapper to communicate with
+// rbac microservice.
+func initRbacWrapper(ctx context.Context, cfg *config.Config) rbac.Rbac {
+	if env, ok := os.LookupEnv("ENV_NAME"); ok && env == "local" {
```

Comment:
All configuration should go through `config.Config`. This lookup adds an auxiliary configuration vector that bypasses `config.Config`.

Reply:
I have removed this block after refactor in a standalone service.

Reply:
for clarification, that I didn't recall about this; the change is made on this PR that decouple the rbac mock service from the main service: https://github.com/podengo-project/idmsvc-backend/pull/178